src/test/scala/cromwell/FileSizeWorkflowSpec.scala

```diff
@@ -0,0 +1,60 @@
+package cromwell
+
+import akka.testkit.EventFilter
+import cromwell.util.SampleWdl
+import wdl4s._
+import wdl4s.values.WdlFloat
+
+/**
+  * Created by thibault on 2/8/16.
+  */
+class FileSizeWorkflowSpec extends CromwellTestkitSpec {
+
+  object FileSize extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """task file_size {
+        |  File input_file
+        |
+        |  command {
+        |    echo "this file is 22 bytes" > created_file
+        |  }
+        |
+        |  output {
+        |    Float input_file_size = size(input_file)
+        |    Float created_file_size = size("created_file")
+        |    Float created_file_size_in_k = size("created_file", "K")
+        |    Float created_file_size_in_m = size("created_file", "M")
+        |    Float created_file_size_in_g = size("created_file", "G")
+        |    Float created_file_size_in_t = size("created_file", "T")
```

Comment:
We supported them because the library (squant) I was using supports them, but if fall back to use the one already in wdl4s then it's not supported. So I don't know I could add support for it in wdl4s but I don't know if it's really worth it since no one actually asked for it